src/constants/dashValue.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+const dashValue = (
+  <span
+    /* eslint-disable-next-line jsx-a11y/aria-role */
+    role="text"
+    aria-label="no value set"
```

Comment:
1) Aria-label should be internationalized. 
2) Please capitalize it when added to translations en.json (No value set.)

Reply:
Also please ask in the stripes channel whether there is already a "Folio way" pattern/component for such no value stuff. As if the label will be "No value set." It may make sense to add it to the stripes components to have internationalization strings in one plase